pkg/api/const.go

```diff
@@ -306,6 +306,22 @@ const (
 	DefaultKubernetesCloudProviderRateLimitBucketWrite = DefaultKubernetesCloudProviderRateLimitBucket
 )
 
+// Azure Stack configures all clusters as if they were large clusters.
+const (
+	DefaultAzureStackKubernetesCloudProviderBackoffRetries       = 6
+	DefaultAzureStackKubernetesCloudProviderBackoffJitter        = 1.0
```

Comment:
Defaults for 1.14.0 onward for cloudProviderBackoffMode is v2
https://github.com/Azure/aks-engine/blob/beb8ad935ff3b4e37c17bbbe129562e28372068a/docs/topics/clusterdefinitions.md#L78

Hence based on Azure Cloud provider code CloudProviderBackoffExponent and CloudProviderBackoffJitter is not needed to be set as it is no op.

Reply:
Code is now deprecated https://github.com/msazurestackworkloads/kubernetes/blob/release-1.14/pkg/cloudprovider/providers/azure/azure.go#L317

Reply:
I would rather keep both Azure and Azure Stack flows consistent.
Worst case seems to be a warning in the logs.